File: EarlGrey.Common/Weather/WindData.cs
namespace EarlGrey.Common.Weather {
    public class WindData {

        /// <summary>Wind speed in KM/H</summary>
        public double? Speed { get; set; }

        /// <summary>Gust speed in KM/H</summary>
        public double? GustSpeed { get; set; }

        /// <summary>Direction to where the wind is GOING in degrees</summary>
        public double? Direction { get; set; }

        /// <summary>Direction string (IE: NW, SW, N, Calm)</summary>
        public string DirectionString => WindDirection.DegreesToCompass(Direction);
    }
}

File: earlgrey.web/src/model/weather/WindData.ts
export default interface WindData {
    /** Speed in KM/H */
    speed: number

    /**Gust speed in KM/H */
    gustSpeed: number

    /**Direction the wind is blowing towards */
    direction: number

    /** Direction cardinal direction (N, NNW, NW, WNW, W, etc.) */
    directionString: string
}